### PR TITLE
WIP Base Scheme eligibility move to new structure

### PR DESCRIPTION
**Pull request by k8mckay**

### CODE REVIEW

File: openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/ESS _PDRS_base_scheme/ESS_PDRS_base_scheme_eligibliity_parameters [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    default_value = True
+    definition_period = ETERNITY
+    label = 'Has implementation already occurred?'
+    metadata = {
+        'display_question' : 'Has your implementation already occurred?'
+    }
+
+
+class Base_implementation_after_1_April_2022(Variable):
+    value_type = input
```
Comment (alisha17) [inline openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/ESS _PDRS_base_scheme/ESS_PDRS_base_scheme_eligibliity_parameters:42]: can you change value_type to date?

Comment (k8mckay) [reply openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/ESS _PDRS_base_scheme/ESS_PDRS_base_scheme_eligibliity_parameters]: This has been changed

File: openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    definition_period = ETERNITY
+    label = 'Is the installation in a centralised system or common area in a Class 2 building?'
+    metadata = {
+        'variable-type' : 'conditional',
+        'display_question' : 'Is the installation in a centralised system or common area in a Class 2 building?'
+    }
+
+
+#Need some Alisha attention on this one
+class HVAC2_climate_zone(Variable):
```
Comment (alisha17) [inline openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters:96]: looks good to me. it should be an enum.

could you remove the inline comments?

File: openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    value_type = Enum #is this an input not an Enum?
+    entity = Building
+    label = "Which climate zone is the End-User equipment installed in, as defined in ESS Table A27?"
+    definition_period = ETERNITY
+    #do I need a default-value here?
```
Comment (alisha17) [inline openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters:101]: We don't have a default value for similar fields. Lets do no default value for now - will change later if required

File: openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters (hunk 2 above)
Comment (k8mckay) [reply openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters:96]: Done!

File: openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters (hunk 3 above)
Comment (k8mckay) [reply openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters:101]: Yep, agreed

**Review by alisha17 — APPROVED** [review]

**Pull request merged**